sky/execution.py

```diff
@@ -1031,7 +1031,7 @@ def serve_up(
         controller_task.best_resources = controller_best_resources
 
         controller_envs = {
-            'SKYPILOT_USER_ID': common_utils.get_user_hash(),
+            'SKYPILOT_USAGE_USER_ID': common_utils.get_user_hash(),
```

Comment:
Yeah, we should discuss... If we could merge `serve-dev` into the master branch in a short time, this branch might be deprecated in a short time too. Since merging `serve` to the current master introduces some conflicts (ports, cluster name, etc.), it might not be worth doing that...

Reply:
I am confused. We are using the `SKYPILOT_USER_ID` in the master branch already https://github.com/skypilot-org/skypilot/blob/bce08e5795e49bc1fac8816c46125d3ed51eccf2/sky/templates/spot-controller.yaml.j2#L41

It seems our first todo is to merge the `serve` branch to master. In that case, we should solve these conflicts, right?